### ISSUE TITLE
autocomplete with sdl2 isn't working

### ISSUE THREAD

**Issue by ellisonch** [intellij-rust/intellij-rust#2591]
Hello,

I can't seem to get intellij-rust's autocomplete working when it comes to using sdl2.  I have everything set up just fine to where it autocompletes internal stuff as well as other external modules, but it just doesn't seem to work with sdl2 specifically:

```rust
extern crate sdl2;

pub fn main() {
	// sdl2:: <--- does autocomplete
	let _x = sdl2::init();
	// _x.  <--- does NOT autocomplete
}
```

The above code runs (and in fact, I can use `_x` for all sorts of stuff that puts nice pictures on the screen).

I'm calling `init()` as described in the sdl2 instructions.  I'm including it as
```toml
[dependencies]
sdl2 = "0.31"
```

It seems as though intellij-rust doesn't understand the type of `sdl2::init()`, so it doesn't know what to offer.  I don't know enough about rust to really understand what's going on (I've written about 10 lines of rust).

**Comment by Undin** [comment]
@ellisonch Hi!
Could you specify your IDE, intellij-rust and rust toolchain versions?
I can't reproduce it with the latest versions of plugin and rust toolchain

**Comment by Undin** [comment]
Oh, I think they are the same as in #2596

**Comment by ellisonch** [comment]
@Undin Yes, the same.  I'm using intellij-rust Version: 0.2.0.2096-181, installed from inside IntelliJ Idea 181.5281.24. All the settings are default. I'm on Windows. My toolchain version is 1.26.2. My whatever-it's-called is stable-x86_64-pc-windows-msvc.
